mobile/client.go

```diff
@@ -57,7 +57,7 @@ func NewProps() *Props { return &Props{make(map[string]interface{})} }
 
 func (ps *Props) String() string                  { return fmt.Sprint(ps.m) }
 func (ps *Props) MarshalJSON() ([]byte, error)    { return json.Marshal(ps.m) }
-func (ps *Props) UnMarshalJSON(data []byte) error { return json.Unmarshal(data, &ps.m) }
+func (ps *Props) UnMarshalJSON(data string) error { return json.Unmarshal([]byte(data), &ps.m) }
```

Comment:
Why are you returning []byte from marshal while unmarshal takes a string?

Reply:
That was a codesmell I did not notice. Tried to fixed it. Thanks 💯